avm/ptn/dev-ops/cicd-agents-and-runners/README.md

```diff
@@ -1328,13 +1328,863 @@ The networking configuration.
 
 - Required: Yes
 - Type: object
+- Type-deciding property: networkType
+
+<h4>The available variants are:</h4>
+
+| Variant | Description |
+| :-- | :-- |
+| [`createNew`](#variant-networkingconfigurationnetworktype-createnew) |  |
+| [`useExisting`](#variant-networkingconfigurationnetworktype-useexisting) |  |
+
+### Variant: `networkingConfiguration.networkType-createNew`
+
```

Comment:
We probably miss a @description on the discriminator every time we have 2 empty lines after the variant header. Should we have this checked? Not for this PR, just again wanted to bring it up (maybe it's already producing a warning)

Reply:
We certainly **do** have a spec that say it's a MUST ([ref](https://azure.github.io/Azure-Verified-Modules/spec/BCPNFR21/)).
I don't think we have a test for it yet though - so I agree it should not be part of this PR, but be implemented. The question, as with the nullable etc. will be whether the test should be enforced or not 😄 But maybe that's something we can decide after the test was implemented and run across the modules to see how many would need fixing. But I guess it's a lot.

Reply:
Hey @AlexanderSehr created #4767